<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>changeInfoDialog</class>
    <widget class="QDialog" name="changeInfoDialog">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>400</width>
                <height>250</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>修改信息</string>
        </property>
        <widget class="QLabel" name="showMsg">
            <property name="geometry">
                <rect>
                    <x>130</x>
                    <y>40</y>
                    <width>141</width>
                    <height>41</height>
                </rect>
            </property>
            <property name="font">
                <font>
                    <family>微软雅黑</family>
                    <pointsize>10</pointsize>
                </font>
            </property>
            <property name="text">
                <string/>
            </property>
            <property name="alignment">
                <set>Qt::AlignCenter</set>
            </property>
        </widget>
        <widget class="QWidget" name="formLayoutWidget">
            <property name="geometry">
                <rect>
                    <x>50</x>
                    <y>100</y>
                    <width>291</width>
                    <height>41</height>
                </rect>
            </property>
            <property name="font">
                <font>
                    <family>微软雅黑</family>
                    <pointsize>10</pointsize>
                </font>
            </property>
            <layout class="QFormLayout" name="formLayout">
                <property name="horizontalSpacing">
                    <number>20</number>
                </property>
                <property name="verticalSpacing">
                    <number>20</number>
                </property>
                <item row="0" column="0">
                    <widget class="QLabel" name="password">
                        <property name="font">
                            <font>
                                <family>微软雅黑</family>
                                <pointsize>10</pointsize>
                            </font>
                        </property>
                        <property name="text">
                            <string/>
                        </property>
                    </widget>
                </item>
                <item row="0" column="1">
                    <widget class="QLineEdit" name="inputMsg">
                        <property name="font">
                            <font>
                                <family>微软雅黑</family>
                                <pointsize>10</pointsize>
                            </font>
                        </property>
                        <property name="echoMode">
                            <enum>QLineEdit::Normal</enum>
                        </property>
                    </widget>
                </item>
            </layout>
        </widget>
        <widget class="QPushButton" name="confirmBtn">
            <property name="geometry">
                <rect>
                    <x>150</x>
                    <y>180</y>
                    <width>93</width>
                    <height>28</height>
                </rect>
            </property>
            <property name="font">
                <font>
                    <family>微软雅黑</family>
                    <pointsize>10</pointsize>
                </font>
            </property>
            <property name="text">
                <string>确认</string>
            </property>
        </widget>
    </widget>
    <resources/>
    <connections/>
</ui>
